src/middlewares/basic-auth.middleware.ts

```diff
@@ -0,0 +1,25 @@
+import { Request, Response, NextFunction } from 'express';
+
+import { ADD_NOTIFICATION_PASSWORD, ADD_NOTIFICATION_USERNAME } from '../config';
+import { isDefined } from '../utils/helpers';
+
+export const basicAuth = (req: Request, res: Response, next: NextFunction) => {
+  if (isDefined(req.get('Authorization'))) {
+    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```

Comment:
Can we remove this eslint disable rule? 

In order to do so, you can assign `req.get('Authorization')` to a variable and then use `isDefined` as a typeguard on the variable value.